src/lib/controller/message_handler.ts

```diff
@@ -81,8 +83,17 @@ export class ControllerMessageHandler {
         return {};
       }
       case ControllerCommand.getProvisioningEntry: {
-        const entry = driver.controller.getProvisioningEntry(message.dsk);
-        return { entry };
+        if (message.dskOrNodeId) {
+          return {
+            entry: driver.controller.getProvisioningEntry(message.dskOrNodeId),
+          };
+        }
+        if (message.dsk) {
+          return { entry: driver.controller.getProvisioningEntry(message.dsk) };
+        }
```

Comment:
Bit more readable I think. 
```suggestion
        const dskOrNodeId = message.dskOrNodeId || message.dsk;

        if (dskOrNodeId) {
          return {
            entry: driver.controller.getProvisioningEntry(dskOrNodeId),
          };
        }
```